edgelet/edgelet-http/Cargo.toml

```diff
@@ -28,24 +29,26 @@ edgelet-utils = { path = "../edgelet-utils" }
 systemd = { path = "../systemd" }
 
 [target.'cfg(unix)'.dependencies]
-hyperlocal = "0.6"
 libc = "0.2"
 nix = "0.11"
 scopeguard = "0.3.3"
 tokio-uds = "0.2"
 
 [target.'cfg(windows)'.dependencies]
 hyper-named-pipe = { path = "../hyper-named-pipe" }
+mio-uds-windows = { git = "https://github.com/damonbarry/mio-uds-windows.git" }
 tokio-named-pipe = { path = "../tokio-named-pipe" }
+tokio-uds-windows = { git = "https://github.com/damonbarry/tokio-uds-windows.git" }
+winapi = { version = "0.3.5", features = ["winsock2"] }
 
 [dev-dependencies]
 lazy_static = "1.0"
+tempfile = "3"
+tempdir = "0.3.7"
 
 edgelet-test-utils = { path = "../edgelet-test-utils" }
 
-[target.'cfg(unix)'.dev-dependencies]
-tempfile = "3"
-
 [target.'cfg(windows)'.dev-dependencies]
 httparse = "1.2"
 rand = "0.4"
+scopeguard = "0.3.3"
```

Comment:
This isn't used, right? You removed it in the `connector.rs` test.

Reply:
I'll check, thanks.